app/controllers/repositories_controller.rb

```diff
@@ -369,7 +369,8 @@ def export_repository_stock_items
       RepositoryStockZipExportJob.perform_later(
         user_id: current_user.id,
         params: {
-          repository_row_ids: row_ids
+          repository_row_ids: row_ids,
+          decimal_precisions: @repository.repository_stock_column.metadata['decimals']
```

Comment:
Don't pass it as params to the job, instead read the current value from inside the job